metadata-ingestion/src/datahub/ingestion/extractor/mce_extractor.py

```diff
@@ -1,27 +1,44 @@
+import time as t
 from typing import Iterable, Union
 
 from datahub.ingestion.api import RecordEnvelope
 from datahub.ingestion.api.common import PipelineContext
 from datahub.ingestion.api.source import Extractor, WorkUnit
 from datahub.ingestion.api.workunit import MetadataWorkUnit, UsageStatsWorkUnit
-from datahub.metadata.com.linkedin.pegasus2avro.mxe import MetadataChangeEvent
+from datahub.metadata.com.linkedin.pegasus2avro.mxe import (
+    MetadataChangeEvent,
+    SystemMetadata,
+)
 from datahub.metadata.schema_classes import UsageAggregationClass
 
 
+def current_milli_time():
+    return round(t.time() * 1000)
```

Comment:
mce_builder.py `get_sys_time` does essentially the same thing - could we just use that?

Reply:
updated!